guacamole-docker/bin/start.sh

```diff
@@ -251,40 +251,97 @@ environment variables or their corresponding Docker secrets by appending _FILE
 to the environment variable, and setting the value to the path of the
 corresponding secret:
 
-    POSTGRES_USER      The user to authenticate as when connecting to
+    POSTGRESQL_USER      The user to authenticate as when connecting to
                        PostgreSQL.
 
-    POSTGRES_PASSWORD  The password to use when authenticating with PostgreSQL
-                       as POSTGRES_USER.
+    POSTGRESQL_PASSWORD  The password to use when authenticating with PostgreSQL
+                       as POSTGRESQL_USER.
 
-    POSTGRES_DATABASE  The name of the PostgreSQL database to use for Guacamole
+    POSTGRESQL_DATABASE  The name of the PostgreSQL database to use for Guacamole
                        authentication.
```

Comment:
Beware that these have become misaligned with the addition of the QL.

Reply:
The indented following lines should be aligned with the lines above, but that's no longer the case with the addition of the extra "QL" in the variable name. 

So the following lines, like "authentication.", should all be indented by 2 additional characters so that they line up again.

Reply:
Oh ok I see and fixed. :) Should be good now. I'll tackle the guacamole-manual after that to reflect the changes.